docs/application/native/guides/connectivity/stc.md

```diff
@@ -137,9 +137,9 @@ To fetch statistics about total network data consumed by system:
    }
    ```
 
```

Comment:
I think we need `<a name="retrieve_data_usage_applications"></a>` here. Please check and add it if it is needed.

Reply:
resolved, now link is working